lua/telescope/_extensions/session-lens/session-lens-library.lua

```diff
@@ -93,7 +93,19 @@ do
     mt_file_entry.display = function(entry)
       -- Revert session files back to paths for easier reading
       -- This affects the display only, a selection still uses the correct file name.
-      local display = path.make_relative((entry.value:gsub("%%", "/")), cwd)
+      local display = ''
+      if vim.fn.has('win32')==1 then
+        local entry_value = entry.value:gsub("%++", ":")
+        entry_value = entry_value:gsub("%%", "/")
+        display = path.make_relative((entry_value), cwd)
+      else
+        display = path.make_relative((entry.value:gsub("%%", "/")), cwd)
+      end
+      -- there problem with telescope path_shorten() for windows
+      if vim.fn.has('win32') then
+        shorten_path=false
+      end
+
       if shorten_path then
         display = utils.path_shorten(display)
       end
```

Comment:
You could also make this a one liner.
```lua
display = not WIN32 and shorten_path and utils.path_shorten(display) or display
```

Reply:
On this line I hesitate to make it simple because there is a problem with the Telescope `path_shorten()` on the Windows and it's not the fault of the plugin. I think `utils.path_shorten (display)` is a feature of this plugin and I hope to fix this once I understand the code on the telescope. But for the moment the change is acceptable.